pkgs/development/python-modules/gpib-ctypes/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pytest-runner
+}:
+
+buildPythonPackage rec {
+  version = "0.3.0";
+  pname = "gpib_ctypes";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-c9l6TNmM4PtbvopnnFi5R1dQ9o3MI39BHHHPSGqfjCY=";
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-runner
+  ];
+
+  pythonImportsCheck = [
+    "gpib_ctypes.gpib"
+  ];
+
+  meta = with lib; {
+    description = "Cross-platform Python bindings for the NI GPIB and linux-gpib C interfaces";
+    longDescription = ''
+      To utilize the installed GPIB library and access GPIB devices, the required shared library
+      files must be located in the following directories:
+
+      For Windows: c:\\windows\\system32\\gpib-32.dll
```

Comment:
```suggestion
```

This isn't need since nix doesn't run on windows, right? Or do you mean wsl?

Reply:
This description will be omitted once `linux-gpib` package is merge at PR #245677 and path is fixed.